packages/server/integrations/gitlab/GitLabServerManager.ts

```diff
@@ -142,10 +145,31 @@ class GitLabServerManager implements TaskIntegrationManager {
     return [noteData, noteError] as const
   }
 
-  async getProjects({teamId}: {teamId: string}) {
+  async getProjectIssues({
+    fullPath,
+    first,
+    includeSubepics,
+    searchQuery,
+    sort,
```

Comment:
In the next PR, these are passed in as one object rather than separate variables